docs/guidelines.rst

```diff
@@ -43,9 +43,15 @@ Create a pull request
   This means that we can NOT accept pull requests from you unless this is done,
   so please don't put the code reviewers at risk and do it anyways.
 
-* Add a changelog entry in ``CHANGES.rst``.
-  Make it clear if it is a fix or a new feature.
+* Add a changelog entry as a `towncrier <https://github.com/hawkowl/towncrier>`_ news entry in the ``news`` directory in the repository root.
+  The entry should make clear what your change is about.
   When in doubt, don't worry about it.
+  The file should follow this naming scheme: ``<issue or PR number>.<changelog type>``.
+  Please reference the issue (on the same Github repository, no cross-references possible here).
```

Comment:
For me it is no problem at all if you use an issue number from a different repository. At least it is fine if you use an issue number from `Product.CMFPlone`. The most important requirement for the number, is that it is unique among any other current news snippets.

Reply:
How can towncrier add a reference to the correct issue if you use a issue number from a different repository?

Reply:
It doesn't add a reference. With the `pyproject.toml` settings that we have in most Plone packages, only the issue number is added. It is not turned into a link.